draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4362,7 +4362,7 @@ Content-Range: exampleunit 11.2-14.3/25
     </tr>
     <tr>
       <td>HEAD</td>
-      <td>Same as GET, but only transfer the status line and header section.</td>
+      <td>Same as GET, but only transfer the status code and header section.</td>
       <td><xref target="HEAD" format="counter"/></td>
```

Comment:
Maybe here we can simplify to "...but do not transfer the response body."?

Reply:
Done, but it raises an interesting question: should HEAD omit the trailer section? I think so, but in theory it could be included w/ h2 and h3.

Reply:
The only stuff sent in a trailer is information generated while sending the body. No server is going to generate and send a body to /dev/null just for the sake of finding trailer info. Apache won't even do that to generate header fields like Content-Encoding.

Reply:
Fair enough.